# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2023/11
# add missing __init__.py files in apps, c_alpha, data, gnn, models, and utils
# in order to allow setup.py.find_packages() to find and install subdirectories
diff -ruN model-angelo-1.0.9/model_angelo/apps/__init__.py model-angelo-1.0.9_add_init_py/model_angelo/apps/__init__.py
--- model-angelo-1.0.9/model_angelo/apps/__init__.py	2023-10-27 16:42:58.000000000 +0200
+++ model-angelo-1.0.9_add_init_py/model_angelo/apps/__init__.py	2023-11-06 11:06:16.212950502 +0100
@@ -0,0 +1 @@
+#
diff -ruN model-angelo-1.0.9/model_angelo/c_alpha/__init__.py model-angelo-1.0.9_add_init_py/model_angelo/c_alpha/__init__.py
--- model-angelo-1.0.9/model_angelo/c_alpha/__init__.py	1970-01-01 01:00:00.000000000 +0100
+++ model-angelo-1.0.9_add_init_py/model_angelo/c_alpha/__init__.py	2023-11-06 11:06:16.212950502 +0100
@@ -0,0 +1 @@
+#
diff -ruN model-angelo-1.0.9/model_angelo/data/__init__.py model-angelo-1.0.9_add_init_py/model_angelo/data/__init__.py
--- model-angelo-1.0.9/model_angelo/data/__init__.py	1970-01-01 01:00:00.000000000 +0100
+++ model-angelo-1.0.9_add_init_py/model_angelo/data/__init__.py	2023-11-06 11:06:16.212950502 +0100
@@ -0,0 +1 @@
+#
diff -ruN model-angelo-1.0.9/model_angelo/gnn/__init__.py model-angelo-1.0.9_add_init_py/model_angelo/gnn/__init__.py
--- model-angelo-1.0.9/model_angelo/gnn/__init__.py	1970-01-01 01:00:00.000000000 +0100
+++ model-angelo-1.0.9_add_init_py/model_angelo/gnn/__init__.py	2023-11-06 11:06:16.212950502 +0100
@@ -0,0 +1 @@
+#
diff -ruN model-angelo-1.0.9/model_angelo/models/__init__.py model-angelo-1.0.9_add_init_py/model_angelo/models/__init__.py
--- model-angelo-1.0.9/model_angelo/models/__init__.py	1970-01-01 01:00:00.000000000 +0100
+++ model-angelo-1.0.9_add_init_py/model_angelo/models/__init__.py	2023-11-06 11:06:16.212950502 +0100
@@ -0,0 +1 @@
+#
diff -ruN model-angelo-1.0.9/model_angelo/utils/__init__.py model-angelo-1.0.9_add_init_py/model_angelo/utils/__init__.py
--- model-angelo-1.0.9/model_angelo/utils/__init__.py	1970-01-01 01:00:00.000000000 +0100
+++ model-angelo-1.0.9_add_init_py/model_angelo/utils/__init__.py	2023-11-06 11:10:29.338107389 +0100
@@ -0,0 +1 @@
+#
